engine/development/deploymentManager/src/main/scala/pl/touk/nussknacker/development/manager/DevelopmentDeploymentManagerProvider.scala

```diff
@@ -109,8 +109,8 @@ class DevelopmentDeploymentManager(actorSystem: ActorSystem)
 
   override def test[T](name: ProcessName, canonicalProcess: CanonicalProcess, scenarioTestData: ScenarioTestData, variableEncoder: Any => T): Future[TestProcess.TestResults[T]] = ???
 
-  override def getFreshProcessState(name: ProcessName): Future[Option[StatusDetails]] =
-    Future.successful(memory.get(name))
+  override protected def getFreshProcessStates(name: ProcessName): Future[List[StatusDetails]] =
```

Comment:
Why right now it's protected?

Reply:
IDE automatically add protected when I override this method. Why it have to be public? `getFreshProcessStates` is not meant to be publically available, `getProcessStates` is for public usage